x-pack/plugins/infra/public/alerting/metric_anomaly/index.ts

```diff
@@ -8,16 +8,15 @@
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { METRIC_ANOMALY_ALERT_TYPE_ID } from '../../../common/alerting/metrics';
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { AlertTypeModel } from '../../../../triggers_actions_ui/public/types';
-import { AlertTypeParams } from '../../../../alerting/common';
+import { AlertTypeModel } from '../../../../triggers_actions_ui/public';
```

Comment:
`import { AlertTypeModel as RuleTypeModel } from '../../../../triggers_actions_ui/public';`

Reply:
I am not sure if we should rename this now? Because we will refactor triggers_actions_ui too, then we can tackle these imports by searching by the usages.